server/filters/get-principle-main-media.js

```diff
@@ -1,10 +1,7 @@
-export default function getPrincipleMainMedia(arr) {
-  const video = arr.find((item) => {
-    return item.type === 'video';
-  });
-  const picture = arr.find((item) => {
-    return item.type === 'picture';
-  });
+export default function getPrincipleMainMedia(arr: []) {
+  const video = arr.find(item => item.type === 'video');
+  const mainPicture = arr.find(item => item.isMain);
```

Comment:
…although without it, I think we'll have two `Picture`s in the main media array and no way to distinguish between them